src/views/v2/list/month-separator.php

```diff
@@ -12,8 +12,15 @@
  * @version 4.9.3
  *
  */
-$month = $this->get( 'month' );
+$date  = $this->get( 'date' );
+$month = date( 'M', $date );
+$year  = date( 'Y', $date );
 ?>
 <div class="tribe-events-calendar-list__separator-month">
-	<time class="tribe-events-calendar-list__separator-month-text tribe-common-b1 tribe-common-b1--bold" datetime="1970-01-01T00:00:00+00:00"><?php echo esc_html( $month ); ?></time>
+	<time
+		class="tribe-events-calendar-list__separator-month-text tribe-common-b1"
```

Comment:
This should be `h7` on mobile and `h6` on desktop. Let's add the `h7` class and override with `@mixin heading-6` in the separator month text class.

Reply:
Why should it be that? The size is for the b1 match the design and they all have the same line height. Would you please expand a bit more on why you think it should be like that?

Reply:
I'm going based off the class names when you inspect the elements. My takeaway from the class names is it gives insight into Matt and Rachel's thoughts when they were designing this. So should any styles regarding `h6` or `h7` change, everything that uses the mixins and classes are updated easily. You're right, the `b1` styles match the design, but if their intention was not for the text to be a body text but a heading with heading styles, then we're not building this in an easily maintainable way. I would check with Matt and Rachel on what they intended here.

Reply:
Also I'm iffy about their heading sizes from `h5` to `h8`, I'm seeing conflicting class names and font sizes. I'm going to poke Matt a bit about this.

Reply:
from the call with design, we'll use heading classes here. Let's define an alt class as needed.

Reply:
I'm adding the `--alt` modifier for now. I see that's a repeated pattern. Maybe we should think about having a generic modifier for `font-weight: normal` that can be used everywhere. `the-prefix--font-normal` or similar.

Reply:
sounds good. I think it's OK for now, we could say the same for the same 3 styles that are applied to every heading (color, font family, and font weight), but we'll also need to balance how easily someone can follow the code. not the highest priority for now either.